migrate_test.go

```diff
@@ -892,6 +894,8 @@ func (m *migrationSequence) bodySequence() []string {
 			v.Body = ioutil.NopCloser(bytes.NewReader(body))
 
 			r = append(r, string(body[:]))
+		} else {
+			r = append(r, "<empty>")
```

Comment:
This makes sure that we don't introduce non-sensical data in the test expectations. The user needs to be made aware when a migration step is empty.